src/tests/ftest/scripts/setup_nodes.sh

```diff
@@ -42,6 +42,10 @@ sudo bash -c "set -ex
 if [ \"\$(ulimit -c)\" != \"unlimited\" ]; then
     echo \"*  soft  core  unlimited\" >> /etc/security/limits.conf
 fi
+if [ \"\$(ulimit -l)\" != \"unlimited\" ]; then
+    echo \"*  soft  memlock  unlimited\" >> /etc/security/limits.conf
+    echo \"*  hard  memlock  unlimited\" >> /etc/security/limits.conf
+fi
```

Comment:
To be perfectly clear (and probably preach to the choir), this only affects our local testing.

I left a question open in the ticket about how production systems become compliant.

Reply:
Who should answer this question? @johannlombardi

Reply:
I'm a little concerned that https://daosio.atlassian.net/browse/DAOS-5061?focusedCommentId=89879 means that this limits change is only needed on clients.  We are applying it to all nodes, including servers.  This has the potential to mask a bug where the server node is not being modified correctly by daos.  That is, everything will work in our CI but will fail in a production (i.e. end-user) environment.

I wonder if making this change needs to be pushed out into the test framework to only be done on clients.